core/src/styles/core/_typography.scss

```diff
@@ -238,3 +238,62 @@ code {
   @include cat-break-word;
   padding: 0.0625rem 0.25rem;
 }
+
+// ----- editorial typography
+
+article{
+
+  font-size: 1rem;
+
+  $cat-head-margin-bottom: 1.5rem; 
+  $cat-body-margin-bottom: 2.5rem;
+
+  :where(h1, h2, h3, h4, h5, h6),
+  .cat-h1,
+  .cat-h2,
+  .cat-h3,
+  .cat-h4,
+  .cat-h5,
+  .cat-h6 {
+    margin-top: 0;
+    margin-bottom: $cat-head-margin-bottom;
+  }
+
+  :where(h1, h2, h3, h4, h5, h6, .cat-h1, .cat-h2, .cat-h3, .cat-h4, .cat-h5, .cat-h6):has(+ h1, + h2, + h3, + h4, + h5, + h6, + .cat-h1, + .cat-h2, + .cat-h3, + .cat-h4, + .cat-h5, + .cat-h6 ) {
```

Comment:
we cannot rely on has without Firefox support :(

Reply:
I know, unfortunately :(. 
Currently looking for a fallback solution for firefox. Our current % of Firefox users for Haiilo is around 4.5%. 

Do you know about any proper fallback?

Reply:
If we wait long enough we might just get Firefox support. 😅  
The are currently working on it and it is supposed to be launched in 2023
https://connect.mozilla.org/t5/ideas/when-is-has-css-selector-going-to-be-fully-implemented-in/idi-p/23794/page/3#comments